### PR TITLE
MGMT-3790: fix deploy-ibip DNS entries

### PR DESCRIPTION
**Pull request by osherdp**
/cc @tsorya @YuviGold 
A better solution will come in the future

### CODE REVIEW

File: discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py [hunk 1]
```diff
@@ -65,7 +65,8 @@ def _terraform_params(self, **kwargs):
                   # TODO change to namespace index
                   "libvirt_secondary_network_if": self.network_conf.libvirt_secondary_network_if,
                   "provisioning_cidr": self.network_conf.provisioning_cidr,
-                  "running": True
+                  "running": True,
+                  "single_node_ip": kwargs.get('single_node_ip', ''),
```
Comment (eranco74) [inline discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py:69]: The terrraform get this param?

Comment (osherdp) [reply discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py:69]: yes
so far it's got changed only on runtime, that's why I needed to add it here

**Review by YuviGold — APPROVED** [review]

**Review by eranco74 — COMMENTED** [review]
/lgtm

**Pull request merged**